src/shared/components/VideoPlayer/VideoPlayer.tsx

```diff
@@ -102,7 +105,7 @@ const VideoPlayer: React.FC<VideoPlayerProps> = ({
         </PlayOverlay>
       )}
       <div data-vjs-player>
-        <video ref={playerRef} className="video-js" />
+        <video ref={playerRef} className="video-js" onTouchStart={handlePlayOverlayClick} />
```

Comment:
We may need some additional handling for that event. The way it works now if I try to scroll down (I need to touch the video surface to do that), my scrolling will trigger the pause. We should be able to pause only if the user taps, not starts touch movement

Reply:
I think I'll revert this change because I'm not sure how to fix this one in a relatively short time.  I'll just create another issue for this(and for the general mobile UX).